.github/workflows/macos.yml

```diff
@@ -53,18 +49,18 @@ jobs:
           echo "NPROC=`sysctl -n hw.ncpu`" >> $GITHUB_ENV
           export TERM="vt100"
           echo "TERM=$TERM" >> $GITHUB_ENV
-
       - name: configure
         run: |
           cd _build
           ../configure --enable-cobc-internal-checks --enable-hardening --prefix /opt/cobol/gnucobol-gcos --exec-prefix /opt/cobol/gnucobol-gcos
+          sed -i '' 's/-undefined suppress//g' config.h
```

Comment:
@GitMensch Also note this "hack" ; `-undefined suppress` has become deprecated on the latest linker shipped with MacOS, and it complains rather loudly (hence the failed tests).
I don't know if we can get rid of this option for every versions, including older ones...

Reply:
Hm, that should be adjusted in configure.ac. Roger changed that line last in 2005:

```diff
-	COB_SHARED_OPT="-dynamiclib"
+	COB_SHARED_OPT="-bundle -flat_namespace -undefined suppress"
```

Can you please check what these mean? Do we need them at all? I'd tend to changing that part to only be used with `rhapsody`  and comment out the `darwin` case.

When adjusting configure.ac: please fix my typo, you will find that before `ototype`.

Reply:
Well, `-dynamiclib` is used to create a "real" dynamic library, while `-bundle` is supposed to create a loadable module, and my (superficial) understanding is that you can not "link" with a loadable module (they're meant to be dlopened).

Just out of curiosity, I'll test if the CI works with `-dynamiclib`.

Reply:
Interestingly, everything works fine if I just remove the specific "darwin" case in `configure.ac`, which sets `COB_SHARED_OPT` to `-shared`. Though I'm not very confident about that (MacOS doc does not mention that option).